mitodl/mitxpro
# make generic resource page in wagtail

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #226 #227 #228 

#### What's this PR do?

- Add generic resource page in wagtail
- Add generic resource template and scss.
- Gives flexibility to marketing team to add any resource page, which will be rendered using same design.

#### How should this be manually tested?

- Add child page of resource page from wagtail.
- Each page should have following properties
  - Title
  - Sub-heading 
  - List of Content (heading and details)
 - Publish it and view live. Page should render using resource template.

#### Screenshots (if appropriate)
**In wagtail:**
<img width="1428" alt="Screen Shot 2019-05-09 at 6 38 40 PM" src="https://user-images.githubusercontent.com/4245618/57457819-b092af80-7289-11e9-8d07-bf4ff01ad431.png">



**Rendered:**
<img width="1275" alt="Screen Shot 2019-05-09 at 2 30 03 PM" src="https://user-images.githubusercontent.com/4245618/57443219-fd18c380-7266-11e9-9ddc-9c31c65c08a3.png">

**More content:**
<img width="1258" alt="Screen Shot 2019-05-09 at 6 43 10 PM" src="https://user-images.githubusercontent.com/4245618/57458184-547c5b00-728a-11e9-9f3a-60eb38a553b5.png">


**Mobile:**
<img width="536" alt="Screen Shot 2019-05-08 at 4 54 00 PM" src="https://user-images.githubusercontent.com/4245618/57443452-87612780-7267-11e9-8ac3-42c67650790f.png">
